### PR TITLE
[typescript/th] finish and up to date for TypeScript's Thai translation

### PR DESCRIPTION
**Pull request by Jedt3D**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by Jedt3D** [comment]
@adambard  this is my 2nd translation. please merge to main repo if it is ok.
Aiming to JavaScript and Python3 soon.

File: th-th/typescript.th.html.markdown [hunk 1]
```diff
@@ -0,0 +1,259 @@
+---
+language: TypeScript
+contributors:
+    - ["Philippe Vlérick", "https://github.com/pvlerick"]
+    - ["Worajedt Sitthidumrong", "https://bitbucket.org/wrj"]
+filename: learntypescript.ts
```
Comment (divayprakash) [inline th-th/typescript.th.html.markdown]: Please change this to 
```
filename: learntypescript-th.ts
```

**Comment by Jedt3D** [comment]
Hi @divayprakash. I've fixed and push to the repo already.

**Comment by adambard** [comment]
Thanks for your contribution!

**Pull request merged**